src/modes/editor/external.cc

```diff
@@ -28,7 +28,7 @@ namespace Astroid {
     editor_started = true;
 
     ustring cmd = ustring::compose (editor_cmd,
-        em->tmpfile_path.c_str ());
+        em->tmpfile_path.c_str (), "''", "-1");
```

Comment:
Why not `''` on both? I assume this works if you have a valid external editor command as well where there are no `%2` and `%3`?